modules/mobile/app/serializers/mobile/v0/claim_serializer.rb

```diff
@@ -10,14 +10,7 @@ class ClaimSerializer
                  :ever_phase_back, :current_phase_back, :requested_decision, :claim_type, :contention_list,
                  :va_representative, :events_timeline
       attribute :updated_at do |data|
-        StatsD.increment('mobile.claims_and_appeals.claim.type',
-                         tags: ["type:#{data.attributes[:claim_type]}"], sample_rate: 1.0)
-
-        if Flipper.enabled?(:mobile_lighthouse_claims, @current_user)
-          data.attributes[:updated_at]
-        else
-          data.attributes[:updated_at].to_time.iso8601
-        end
+        data.attributes[:updated_at]
```

Comment:
Nice. I like to see this go away. I wondered in your PR if there was a better way.

Reply:
yeah, probably should've been done earlier but I didn't want fix code that was going to be deleted soon enough anyways. but it became a problem